docs/storagedriver/aufs-driver.md

```diff
@@ -0,0 +1,191 @@
+<!--[metadata]> +++
+title = "AUFS storage driver in practice"
+description = "Learn how
+to optimize your use of AUFS driver."
+keywords = ["container, storage, driver, AUFS "]
+[menu.main]
+parent = "mn_storage_docker"
++++ <![end-metadata]-->
+
+# Docker and AUFS in practice
+
+AUFS was the first storage driver in use with Docker. As a result, it has a long and close history with Docker, is very stable, has a lot of real-world deployments, and has strong community support.  AUFS has several features that make it a good choice for Docker. These features enable:
+
+- Fast container startup times.
+- Efficient use of storage.
+- Efficient use of memory.
+
+Despite its capabilities and long history with Docker, some Linux distributions do not support AUFS. This is usually because AUFS is not included in the mainline (upstream) Linux kernel. 
+
+Let's take a look at some AUFS features and how they relate to Docker.
+
+## Image layering and sharing with AUFS
+
+AUFS is a *unification filesystem*. This means that it takes multiple directories on a single Linux host, stacks them on top of each other, and provides a single unified view. To achieve this, AUFS uses *union mount*.
+
+AUFS stacks multiple directories and exposes them as a unified view through a single mount point. All of the directories being stacked, as well as the union mount point, must all exist on the same Linux host. AUFS refers to each directory that it stacks as a *branch*.
+
+Within Docker, AUFS union mounts enable image layering. The AUFS storage driver implements Docker image layers using this union mount system. AUFS branches correspond to Docker image layers. The diagram below shows a Docker container based on the `ubuntu:latest` image.
+
+![](http://farm1.staticflickr.com/651/22053243895_8fde0f13e2_b.jpg)
```

Comment:
I take it these will need to be replaced with the local paths

If these images were created in a vector format, Can we have the source-files for those files added to the repository as well (e.g. a SVG or gliffy-JSON), just in case they need to be updated in the future.

Perhaps even replacing the jpg's with SVG